rcl/include/rcl/arguments.h

```diff
@@ -42,7 +42,8 @@ typedef struct rcl_arguments_t
 #define RCL_PARAM_FILE_FLAG "--params-file"
 #define RCL_REMAP_FLAG "--remap"
 #define RCL_SHORT_REMAP_FLAG "-r"
-#define RCL_SECURITY_CONTEXT_FLAG "--security-context"
+#define RCL_ENCLAVE_FLAG "--enclave"
```

Comment:
Same here, especially in the long version of the flag.

Reply:
this set of PRs is applying the renaming as described in the final state of the [design doc](https://github.com/ros2/design/pull/274).
Please comment there for suggesting changes to flag names to keep discussion in a single place